### ISSUE TITLE
formspree.io integration

### ISSUE THREAD

**Issue by keilrosen** [keilrosen/webshit-main#1]
I need to set up a form for this (yknow, makes sense theres a contact page), and I'm using [formspree.io](url), but I don't have that much experience with forms so I'm really not sure how to set it up, especially because I don't have a designated text box, because I prefer my current layout, where you can select the issue you're reaching out about and I contact you from there, or they can reach out on social media.

The code I currently have reads as this: 
```
<form
  action="https://formspree.io/f/xpzbkalb"
  method="POST"
>
```

thats liked a focused view, so I can include the token, here's all the code:

```
<form
  action="https://formspree.io/f/xpzbkalb"
  method="POST"
>
  <label>
    Your email:
    <input type="email" name="email">
  </label>
  <label>
    Your message:
    <textarea name="message"></textarea>
  </label>
  <!-- your other form fields go here -->
  <button type="submit">Send</button>
</form>

```

TL;DR, I can't really figure out how I integrate this, and I need your help -- thank you kindly
